proto/ibc/core/channel/v1/tx.proto

```diff
@@ -91,7 +92,9 @@ message MsgChannelOpenTry {
 }
 
 // MsgChannelOpenTryResponse defines the Msg/ChannelOpenTry response type.
-message MsgChannelOpenTryResponse {}
+message MsgChannelOpenTryResponse {
+  string version = 2 [(gogoproto.moretags) = "yaml:\"version\""];
```

Comment:
Is there a reason not to use `1` for the field number?

Reply:
I changed field number to `1` in new commit.